hacs.json

```diff
@@ -1,10 +1,8 @@
-
 {
     "name": "vacuum-styj02ym",
     "country": ["FR", "PL"],
     "domains": ["vacuum"],
-    "homeassistant": "0.109.0",
-    "iot_class": "local_poll",
-    "zip_release": true,
-    "filename": "miio2.zip"
```

Comment:
Can you please do not remove those lines? I have already done a missing release.
```    
    "zip_release": true,
    "filename": "miio2.zip"	
```